store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -137,6 +141,31 @@ public void initialise(final Schema schema, final StoreProperties properties)
         TableUtils.ensureTableExists(this);
     }
 
+    @Override
+    public void validateSchemas() {
+        super.validateSchemas();
+
+        final ValidationResult validationResult = new ValidationResult();
```

Comment:
In addition to checking the property/type serialisers please can you also check the vertexSerialiser.

e.g:

```java
if (!(getSchema().getVertexSerialiser() instanceof ToBytesSerialiser)) {
    // add error
}
```